CRM/Contact/Form/Task/EmailTrait.php

```diff
@@ -173,27 +172,13 @@ public function buildQuickForm() {
       'multiple' => TRUE,
     ]);
 
-    if ($to->getValue()) {
```

Comment:
This to stuff is separately in https://github.com/civicrm/civicrm-core/pull/21741